doc/feature/pointDataEvolution.md

```diff
@@ -0,0 +1,123 @@
+Evolution of data at a point over time
+======================================
+
+One of the features that would make the tool extremely useful for scientists, is
+the ability to obtain the evolution of the data at a specific point over time.
+For example, they could load a current report and compute the LFP for a specific
+target, and once they have their full volume, they could focus on a specific
+point of the scene (e.g. a voxel), and obtain a 2D curve showing the evolution
+of voltage at that point for the whole available time span.
+
+## Requirements
+
+* Users should be able to specify the coordinates of a 3D point as input.
+* As output, users will obtain a series of values corresponding to the evolution
+  of the data at that specific point over time, one value per simulation frame,
+  using always the default report dt.
+* The computation should be independent from the voxelization step, as the
+  point for which the values will be computed should not be limited to discrete
+  voxels, but a continuous 3D space.
+* This feature is not related to the (online) streaming of values. It is just an
+  additional (offline) step that is done after loading and processing the data,
+  to report the information in a more localized way.
+
+## Implementation
+
+The core of the computation is already present in Fivox in the form of functors,
+that take a 3D point as a parameter and, based on the use case and the different
+types of functors available, return a value that is the result of evaluating the
+existing events.
+
+To make the computation of these values easy from a user point of view, we need
+to expose the usage of the functors in a separate step as an independent
+application.
+
+For that reason, the 'voxelize' app will need to be refactored to extract the
+common parts into a superclass that will be used as base for all the
+applications.
+
+## API
+
+    class CommandLineApplication
+    {
+        virtual program_options getHelp(); // all generic options (including volume URI details)
+        Vector2ui getFrameRange();
+        ImageSourcePtr getImageSource();
+    }
+
+    class Voxelize : CommandLineApplication
+    {
+        program_options getHelp(); // adding voxelize-specific options
+        sample(...); // existing function
+    }
+
+    class SamplePoint : CommandLineApplication
+    {
+        program_options getHelp(); // adding application-specific options (3d point)
+        write( Vector3f point, Vector2ui frameRange, string filename );
+    }
+
+## Usage
+
+The usage of the C++ app would be as:
+
+    dataEvolution --volume "fivox://BlueConfig?target=Column&dt=0.1" --point [x, y, z] --frame[s] "0 10" [or alternatively, --time[s] "0 100"] --output data.txt
```

Comment:
Are you calling it dataEvolution or data-evoluation in the end. This is not consistent with the text in the first issue. By the way, I don't like the name very much...